Contributing.md

```diff
@@ -18,6 +18,18 @@ As an organization contributor, you can merge other people's pull requests, or o
 
 This project is not continuously deployed, there is space for debate after review too. Offering everyone the chance to revert, or make an amending pull request. If it feels right, merge.
 
+#### Bug fixes and Release cycles
+
+We maintain two major branches: `master` and `develop`.
+
+`master` is for working on the current release, so any bug fixes or documentation spelling fixes should be merged into this branch.
+
+`develop` is where we stage work for the *next* release, i.e. breaking API changes and related documentation updates. Contributors should gently encourage new pull-requests to point to the appropriate branch, and to rebase onto that branch if necessary.
+
+When a new version is ready to be released, please create a pull request to merge `develop` into `master`, named something like "Release 10.0". Then we can have some final discussion before we merge it into `master` and push the release out to the public.
+
+Since `develop` is a *shared* branch, it is important not to ever rebase this branch onto `master`. If a bug fix is applied to `master` it can be merged into `develop` using good old simple `git checkout develop && git merge master`. Yes this will clutter the history a little bit, but it also provides important context to know how/when a patch was applied. Merge commits can be considered necessary historical data, not warts on an idealized history graph.
```

Comment:
Not to be the "you should write a blog post about this" guy, but you should think about writing a blog post about this!

Reply:
I really like this last paragraph too because it stresses the importance of `merge` vs `rebase` on the development branch and this is something I did not know 👍

Reply:
I don't want to disagree - you're probably right - but whats the upside of putting a bugfix into master and then merging that into develop instead of creating the bugfix commit in develop and then cherry-picking that commit onto master?

Reply:
@AndrewSB There's nothing *wrong* with that, per se, but the downside is that you would have two commits that do the same thing, but that have different SHAs.  In the history, after merging, you would have both commits, but one would become effectively empty.

A merge commit would show "on this day, these commits were merged in from master", which is itself useful information.

Reply:
that makes sense, thanks @colinta 😄